subrootpaths.go

```diff
@@ -133,8 +127,15 @@ func GetSubrootPaths(squareSize uint, idxStart uint, shareCount uint) ([][][]int
 		return nil, srpInvalidShareCount
 	}
 
+	isn := idxStart + shareCount
```

Comment:
[question] what does isn stand for? Since it isn't immediately obvious, proposal to rename to something more verbose

Reply:
It's just a variable combo of idxStart and shareCount: isn

I don't really know what contextual name to give it :-)